src/main/cli/process.ts

```diff
@@ -0,0 +1,142 @@
+// ==LICENSE-BEGIN==
+// Copyright 2017 European Digital Reading Lab. All rights reserved.
+// Licensed to the Readium Foundation under one or more contributor license agreements.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file exposed on Github (readium) in the project repository.
+// ==LICENSE-END==
+
+import * as debug_ from "debug";
+import { app, dialog } from "electron";
+import * as glob from "glob";
+import { EOL } from "os";
+import * as path from "path";
+import { _APP_NAME, _APP_VERSION, _PACKAGING } from "readium-desktop/preprocessor-directives";
+import * as yargs from "yargs";
+
+import { cli_, cliImport, cliOpds, cliRead } from "./commandLine";
+
+// Logger
+const debug = debug_("readium-desktop:cli");
+
+export function cli(mainFct: () => void) {
+
+    yargs
+        .scriptName(_APP_NAME)
+        .version(_APP_VERSION)
+        .usage("$0 <cmd> [args]")
+        .command("opds <title> <url>",
+            "import opds feed",
+            (y) =>
+                y.positional("title", {
+                    describe: "title opds feed",
+                    type: "string",
+                })
+                .positional("url", {
+                    describe: "url opds feed",
+                    type: "string",
+                })
+            ,
+            (argv) => {
+                const promise = cliOpds(argv.title, argv.url);
+                promise.then((isValid) => {
+                    if (isValid) {
+                        app.exit(0);
+                        return;
```

Comment:
Just a thought (tell me what you think): as a CLI user, if the operation succeeds I get a blank response (no information in the console). Should we output a shell message like `process.stdout.write("OPDS import done." + EOL);`?